content/topics/clean-code/python/_index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Clean Code for Python
+ready: True
+---
+
+# Clean Code for Python
+
+## Tabs or Spaces
+
+`4 spaces == 1 tab`
+
+Spaces are the preferred indentation method.
+Tabs should be used solely to remain consistent with code that is already indented with tabs.
+Python 3 **DISALLOWS** mixing the use of tabs and spaces for indentation.
+Python 2 code indented with a mixture of tabs and spaces should be converted to using spaces exclusively.
```

Comment:
Just use spaces. I dont even think any serious open source project uses tabs in Python code.